test_autoflake.py

```diff
@@ -1224,6 +1224,43 @@ def test_exclude(self):
         finally:
             shutil.rmtree(temp_directory)
 
+    def test_with_ignore_init_module_imports_flag(self):
+        # Need a temp directory in order to specify file name as __init__.py
+        temp_directory = tempfile.mkdtemp(dir='.')
+        try:
+            with open(
+                 os.path.join(temp_directory, '__init__.py'), 'w') as output:
+                output.write('import re\n')
+
+            p = subprocess.Popen(
+                list(AUTOFLAKE_COMMAND) +
+                [temp_directory + '/__init__.py',
+                    '--ignore-init-module-imports'],
+                stdout=subprocess.PIPE)
+            result = p.communicate()[0].decode('utf-8')
+
+            self.assertNotIn('import re', result)
+        finally:
+            shutil.rmtree(temp_directory)
+
+    def test_without_ignore_init_module_imports_flag(self):
+        # Need a temp directory in order to specify file name as __init__.py
+        temp_directory = tempfile.mkdtemp(dir='.')
+        try:
+            with open(
+                 os.path.join(temp_directory, '__init__.py'), 'w') as output:
```

Comment:
again, strange placement of the first argument, but the problem goes away if creating a variable like above

Reply:
Yep, fixed!